BSPConvert.Lib/Source/MaterialConverter.cs

```diff
@@ -210,19 +213,21 @@ private string GenerateUnlitVMT(Shader shader)
 
 		private string GenerateLitVMT(Shader shader)
 		{
+			var lightmapped = true;
 			var sb = new StringBuilder();
 			sb.AppendLine("LightmappedGeneric");
 			sb.AppendLine("{");
 
-			AppendShaderParameters(sb, shader);
+			AppendShaderParameters(sb, shader, lightmapped);
 
 			sb.AppendLine("}");
 
 			return sb.ToString();
 		}
 
-		private void AppendShaderParameters(StringBuilder sb, Shader shader)
+		private void AppendShaderParameters(StringBuilder sb, Shader shader, bool lightmapped)
 		{
+			var shadowCorrection = Math.Round(255f / smoothShadows);
```

Comment:
This should be only be defined in the scope that uses it (inside of the `else if (lightmapped && smoothShadows > 1)` block on line 256`)